javascript/javascript1/week4/Voice assistant/voice-assistant.js

```diff
@@ -1,28 +1,122 @@
+"use strict";
 
-const userInformation = {
-    name: '',
-    toDoList: [],
-    day: '',
-    favoriteDish: '',
-    myCalendar: []
-};
+let names = [];
+let whoAmI = 'What is my name';
+let day = 'What day is it today';
+let todo = [];
+let myTodoList = 'What is on my todo';
+let favoriteDish = [];
+let event = [];
+let eventList = 'What am I doing this week';
 
-  //Introducing myself
-  function getReply(command) {
-    if(command.toString().toLowerCase() === 'hello my name is Benjamin'.toLowerCase()){
-    return ('nice to meet you Benjamin');
-    } else if(command.toString().toLowerCase() === 'what is my name'.toLowerCase()){
-        if(userInformation.name === undefined){
-            return `You have not introduced yourself`;
-        } else {
-        return `${userInformation.name}`;
-     } 
+function getReply(command) {
+
+    // 1. To save the person's name and returning nice to meet you along the person's name
+    if (command.startsWith('Hello') && command.includes('name')) {
```

Comment:
What would happen  if I call this function twice like bellow? Can you spot the issue?
`console.log(getReply('Hello my name is Benjamin'));
console.log(getReply('Hello my name is Benjamin'));`